src/Directory.Build.targets

```diff
@@ -0,0 +1,14 @@
+<Project>
+  <PropertyGroup>
+    <BuildDependsOn>EmbedPaketFiles;$(BuildDependsOn)</BuildDependsOn>
+  </PropertyGroup>
+  <Target Name="EmbedPaketFiles">
+    <ItemGroup>
+      <EmbeddedFiles Include="$([System.IO.Path]::GetFullPath('%(Compile.Identity)'))"
+        Condition="$([System.String]::Copy('%(Identity)').Contains('paket-files'))" />
+    </ItemGroup>
+    <CreateProperty Value="@(EmbeddedFiles)">  
+      <Output TaskParameter="Value" PropertyName="embed" />  
+    </CreateProperty>
+  </Target>
+</Project>
```

Comment:
This entire file can be replaced with simply another PackageReference in `Directory.Build.props`. 
http://www.nuget.org/packages/SourceLink.Embed.PaketFiles/2.1.2
``` xml
<PackageReference Include="SourceLink.Embed.PaketFiles" Version="2.1.2" PrivateAssets="all" />
```